### PR TITLE
Needles for Yast module: yast2-rmt

### PR DESCRIPTION
**Pull request by JRivrain**
Progess Ticket : https://progress.opensuse.org/issues/25800
Related PR : https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6641

### CODE REVIEW

**Comment by okurz** [comment]
If you don't mind could you please delete the space in before of the colon ":"? Sorry, being a pedantic QA engineer ;)

File: yast2_rmt-config_written_successfully-20190124.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "xpos": 383,
+      "ypos": 320,
+      "width": 66,
+      "height": 16,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "config_written_successfully"
```
Comment (okurz) [inline yast2_rmt-config_written_successfully-20190124.json]: that tag sounds a bit too generic. Please use a prefix as in the next needle

Comment (JRivrain) [reply yast2_rmt-config_written_successfully-20190124.json]: Right, I missed that one.

Comment (JRivrain) [reply yast2_rmt-config_written_successfully-20190124.json]: Done

**Pull request merged**